packages/nouns-contracts/contracts/client-incentives/NounsClientToken.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-3.0
+
+/*********************************
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░██░░░████░░██░░░████░░░ *
+ * ░░██████░░░████████░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░██░░██░░░████░░██░░░████░░░ *
+ * ░░░░░░█████████░░█████████░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ *
+ *********************************/
+
+pragma solidity ^0.8.19;
+
+import { ERC721 } from '@openzeppelin/contracts-v5/token/ERC721/ERC721.sol';
+import { Ownable } from '@openzeppelin/contracts-v5/access/Ownable.sol';
+import { INounsClientTokenTypes } from './INounsClientTokenTypes.sol';
+import { INounsClientTokenDescriptor } from './INounsClientTokenDescriptor.sol';
+
+contract NounsClientToken is INounsClientTokenTypes, ERC721('Nouns Client Token', 'NOUNSCLIENT'), Ownable {
+    /// @dev This is a ERC-7201 storage location, calculated using:
+    /// @dev keccak256(abi.encode(uint256(keccak256("nounsclienttoken")) - 1)) & ~bytes32(uint256(0xff));
+    bytes32 private constant _STORAGE_LOCATION = 0x8cf5ce6e8ba000976223217bb8fd99e6473b9f0c4b7adc07d894a8f739887e00;
```

Comment:
@eladmallel why not do `_STORAGE_LOCATION = keccak256(...)` ?

Reply:
if you do the calculation you get this compilation error:
```
Error (7615): Only direct number constants and references to such constants are supported by inline assembly.
```

Because we're using it in assembly to set the storage slot, it has to be a hard-coded value.

Makes sense?